.php_cs

```diff
@@ -1,29 +1,28 @@
 #!/usr/bin/env php
 <?php
-
-$finder = Symfony\CS\Finder::create()
-    ->ignoreDotFiles(true)
-    ->ignoreVCS(true)
-    ->exclude('app/config')
-    ->exclude('app/data')
-    ->exclude('app/Resources')
-    ->exclude('var')
-    ->exclude('vendor')
-    ->exclude('web/bundles')
-    ->exclude('web/css')
-    ->exclude('web/fonts')
-    ->exclude('web/js')
-    ->notPath('web/config.php')
-    ->in(__DIR__)
-;
-
-return Symfony\CS\Config::create()
-    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
-    ->fixers([
-        '-psr0', // Ignore Tests\ namespace prefix mismatch with tests/ directory
-        'ordered_use',
-        'phpdoc_order',
-        'short_array_syntax',
+return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+        '@Symfony' => true,
+        '@Symfony:risky' => true,
+        'array_syntax' => ['syntax' => 'short'],
+        'ordered_imports' => true,
+        'phpdoc_order' => true,
```

Comment:
should probably add header for 

    /*
     * This file is part of the Symfony package.
     *
     * (c) Fabien Potencier <fabien@symfony.com>
     *
     * For the full copyright and license information, please view the LICENSE
     * file that was distributed with this source code.
     */